src/Projection/ProjectionValidatorException.php

```diff
@@ -0,0 +1,14 @@
+<?php declare(strict_types=1);
+
+namespace Comquer\Projection;
+
+use RuntimeException;
+
+class ProjectionValidatorException extends RuntimeException
```

Comment:
Consider adding final keyword, I cant see a reason why this exception might be extended.